docs/manage-jobs-in-gravitino.md

```diff
@@ -0,0 +1,583 @@
+     http://localhost:8090/api/metalakes/test/jobs/templates
+     
+Or using query parameter "details=true" to get more details of the job templates:
+
+curl -X GET -H "Accept: application/vnd.gravitino.v1+json" \
+     http://localhost:8090/api/metalakes/test/jobs/templates?details=true
+```
+
+</TabItem>
+<TabItem value="java" label="Java">
+
+```java
+    GravitinoClient client = ...;
+    List<JobTemplate> detailedJobTemplates = client.listJobTemplates();
+```
+
+</TabItem>
+<TabItem value="python" label="Python">
+
+```python
+    client = GravitinoClient(...)
+    detailed_job_templates = client.list_job_templates()
+```
+
+</TabItem>
+</Tabs>
+
+### Get a registered job template by name
+
+You can get a registered job template by its name using the REST API or the Java and Python SDKs.
+
+<Tabs groupId='language' queryString>
+<TabItem value="shell" label="Shell">
+
+```shell
+curl -X GET -H "Accept: application/vnd.gravitino.v1+json" \
+     http://localhost:8090/api/metalakes/test/jobs/templates/my_shell_job_template
+```
+
+</TabItem>
+<TabItem value="java" label="Java">
+
+```java
+    GravitinoClient client = ...;
+    JobTemplate jobTemplate = client.getJobTemplate("my_shell_job_template");
+```
+
+</TabItem>
+<TabItem value="python" label="Python">
+
+```python
+    client = GravitinoClient(...)
+    job_template = client.get_job_template("my_shell_job_template")
+```
+
+</TabItem>
+</Tabs>
+
+### Delete a registered job template by name
+
+You can delete a registered job template by its name using the REST API or the Java and Python SDKs.
+
+Note that deleting a job template will also delete all the jobs that are using this job template.
+If there are queued, started, or cancelling jobs that are using this job template, the deletion 
+will fail with an `InUseException` error.
+
+<Tabs groupId='language' queryString>
+
+<TabItem value="shell" label="Shell">
+
+```shell
+curl -X DELETE -H "Accept: application/vnd.gravitino.v1+json" \
+     http://localhost:8090/api/metalakes/test/jobs/templates/my_shell_job_template
+```
+
+</TabItem>
+<TabItem value="java" label="Java">
+
+```java
+    GravitinoClient client = ...;
+    client.deleteJobTemplate("my_shell_job_template");
+```
+
+</TabItem>
+<TabItem value="python" label="Python">
+
+```python
+    client = GravitinoClient(...)
+    client.delete_job_template("my_shell_job_template")
+```
+
+</TabItem>
+</Tabs>
+
+### Run a job based on a job template
+
+To run a job based on the registered job template, you can use the REST API or the Java and Python SDKs. 
+When running a job, you need to provide the job template name and the parameters to replace the
+placeholders in the job template.
+
+Gravitino leverages the job executor to run the job, so you need to specify the job executor 
+through configuration "gravitino.job.executor". By default, it is set to "local", which means 
+the job will be launched in the local process. Note that the local job executor is only for 
```

Comment:
The job will be lauched as a local process within the same machine that runs the Gravitino server.